### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by huynguyennovem**
**What this PR does / why we need it**:
Correct some words spelling for reading more easily.
File changed:
- CONTRIBUTING.md
- docs/chart_best_practices/rbac.md
- docs/chart_template_guide/functions_and_pipelines.md
- docs/chart_template_guide/helm_ignore_file.md
- docs/charts.md
- docs/charts_hooks.md
- docs/charts_tips_and_tricks.md
- docs/helm/helm_verify.md

**Special notes for your reviewer**:
N/A

**If applicable**:
N/A

Co-Authored-By: Nguyen Van Trung [trungnvfet@outlook.com](mailto:trungnvfet@outlook.com)
Signed-off-by: Nguyen Quang Huy [huynq0911@gmail.com](mailto:huynq0911@gmail.com)

### CODE REVIEW

File: docs/chart_best_practices/rbac.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ RBAC resources are:
 
 ## YAML Configuration
 
-RBAC and ServiceAccount configuration should happen under separate keys. They are separate things. Splitting these two concepts out in the YAML disambiguates them and make this clearer.
+RBAC and ServiceAccount configuration should happen under separate keys. They are separate things. Splitting these two concepts out in the YAML disambiguation them and make this clearer.
```
Comment (hickeyma) [inline docs/chart_best_practices/rbac.md]: disambiguates is the correct word grammatically.

**Review by hickeyma — CHANGES_REQUESTED** [review]
Small nit change

File: docs/chart_best_practices/rbac.md (hunk 1 above)
Comment (huynq0911) [reply docs/chart_best_practices/rbac.md]: I see

**Review by hickeyma — APPROVED** [review]
LGTM, thanks.

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**